parts/k8s/cloud-init/artifacts/cse_helpers.sh

```diff
@@ -23,6 +23,7 @@ if ! [[ $(echo -n "$PRIVATE_IP" | grep -c '^') == 1 ]]; then
 fi
 export PRIVATE_IP
 APT_CACHE_DIR=/var/cache/apt/archives/
+PERMANENT_CACHE_DIR=/root/
```

Comment:
Is there a reason this is in `/root/` directly and not, say, in  `/root/aptcache/` or something like that.

I would think you would want the cache directory to be unique (that is, it only contains cache data) and not be conflated with potentially other information.

PS - `/root/` is the root user home directory

Reply:
We can do that, `/root`/ is kind of a proof of concept. I wanted to get this to a place where it functionally works (it does).

Reply:
addressed